### PR TITLE
feat(llm): rename category

### PR DESCRIPTION
**Pull request by bene2k1**
### Your checklist for this pull request

- [ ] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: ai-data/managed-inference/quickstart.mdx [hunk 1]
```diff
@@ -1,38 +1,38 @@
...
+* **Easy deployment**: Deploy state-of-the-art open weights LLMs with just a few clicks. Scaleway Managed Inference provides a simple and intuitive interface for generating dedicated endpoints.
 * **Security**: Scaleway provides a secure environment for running your models. Our platform is built on top of a secure architecture, and we use state-of-the-art cloud security.
 * **Complete data privacy**: No storage or third-party access to your data (prompt or responses), to ensure it remains exclusively yours.
-* **Auto-scaling (coming soon)**: Scaleway LLM Inference automatically scales your instances based on demand, ensuring that your models are always available and responsive.
+* **Auto-scaling (coming soon)**: Scaleway Managed Inference automatically scales your instances based on demand, ensuring that your models are always available and responsive.
 
 <Message type="important">
   This service is in private beta. [Specific terms and conditions](https://www.scaleway.com/en/contracts/) apply.
 </Message>
```
Comment (bene2k1) [inline ai-data/managed-inference/quickstart.mdx:27]: @tgenaitay shall we remove this?

File: ai-data/managed-inference/how-to/delete-deployment.mdx [hunk 2]
```diff
@@ -1,10 +1,10 @@
 ---
 meta:
-  title: How to delete an LLM Inference deployment
-  description: This page explains how to delete an LLM Inference deployment
+  title: How to delete an Managed Inference deployment
+  description: This page explains how to delete an Managed Inference deployment
```
Comment (nerda-codes) [inline ai-data/managed-inference/how-to/delete-deployment.mdx:4]:
```suggestion
  title: How to delete a Managed Inference deployment
  description: This page explains how to delete a Managed Inference deployment
```

File: ai-data/managed-inference/how-to/delete-deployment.mdx [hunk 3]
```diff
@@ -1,10 +1,10 @@
...
 content:
-  h1: How to delete an LLM Inference deployment
-  paragraph: This page explains how to delete an LLM Inference deployment
+  h1: How to delete an Managed Inference deployment
+  paragraph: This page explains how to delete an Managed Inference deployment
```
Comment (nerda-codes) [inline ai-data/managed-inference/how-to/delete-deployment.mdx:7]:
```suggestion
  h1: How to delete a Managed Inference deployment
  paragraph: This page explains how to delete a Managed Inference deployment
```

File: ai-data/managed-inference/how-to/delete-deployment.mdx [hunk 4]
```diff
@@ -18,10 +18,10 @@ Once you have finished your inference tasks you can delete your deployment. This
 <Macro id="requirements" />
 
   - A Scaleway account logged into the [console](https://console.scaleway.com)
-  - An [LLM Inference deployment](/ai-data/llm-inference/quickstart/)
+  - An [Managed Inference deployment](/ai-data/managed-inference/quickstart/)
```
Comment (nerda-codes) [inline ai-data/managed-inference/how-to/delete-deployment.mdx:21]:
```suggestion
  - A [Managed Inference deployment](/ai-data/managed-inference/quickstart/)
```

File: ai-data/managed-inference/how-to/manage-allowed-ips.mdx [hunk 5]
```diff
@@ -1,32 +1,32 @@
...
 ---
 
-Allowed IPs restrict the IPs allowed to access your LLM Inference endpoints. In the absence of allowed IPs, all IP addresses are allowed by default.
+Allowed IPs restrict the IPs allowed to access your Managed Inference endpoints. In the absence of allowed IPs, all IP addresses are allowed by default.
 
 <Macro id="requirements" />
 
   - A Scaleway account logged into the [console](https://console.scaleway.com)
-  - An [LLM Inference deployment](/ai-data/llm-inference/quickstart/)
+  - An [Managed Inference deployment](/ai-data/managed-inference/quickstart/)
```
Comment (nerda-codes) [inline ai-data/managed-inference/how-to/manage-allowed-ips.mdx:21]:
```suggestion
  - A [Managed Inference deployment](/ai-data/managed-inference/quickstart/)
```

File: ai-data/managed-inference/how-to/monitor-deployment.mdx [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+---
+
+This documentation page shows you how to monitor your Managed Inference deployment with [Cockpit](/observability/cockpit/quickstart/).
+
+<Macro id="cockpit-update-dashboards" />
+
+<Macro id="requirements" />
+
+  - A Scaleway account logged into the [console](https://console.scaleway.com)
+  - An [Managed Inference deployment](/ai-data/managed-inference/quickstart/)
```
Comment (nerda-codes) [inline ai-data/managed-inference/how-to/monitor-deployment.mdx:23]:
```suggestion
  title: How to monitor a Managed Inference deployment
  description: This page explains how to monitor a Managed Inference deployment
content:
  h1: How to monitor a Managed Inference deployment
  paragraph: This page explains how to monitor a Managed Inference deployment
tags: ai monitoring
dates:
  validation: 2024-03-06
  posted: 2024-03-06
categories:
  - ai-data
---

This documentation page shows you how to monitor your Managed Inference deployment with [Cockpit](/observability/cockpit/quickstart/).

<Macro id="cockpit-update-dashboards" />

<Macro id="requirements" />

  - A Scaleway account logged into the [console](https://console.scaleway.com)
  - A [Managed Inference deployment](/ai-data/managed-inference/quickstart/)
```

File: ai-data/managed-inference/index.mdx [hunk 7]
```diff
@@ -20,42 +20,42 @@ meta:
         icon="rocket"
         description="Learn how to create, connect to, and delete an LLM endpoint in a few steps."
```
Comment (nerda-codes) [inline ai-data/managed-inference/index.mdx:21]:
```suggestion
        description="Learn how to create, connect to, and delete a Managed Inference endpoint in a few steps."
```

File: ai-data/managed-inference/index.mdx [hunk 8]
```diff
@@ -20,42 +20,42 @@ meta:
...
-        description="Core concepts that give you a better understanding of Scaleway LLM Inference."
+        description="Core concepts that give you a better understanding of Scaleway Managed Inference."
         label="View Concepts"
-        url="/ai-data/llm-inference/concepts/"
+        url="/ai-data/managed-inference/concepts/"
     />
     <SummaryCard
         title="How-tos"
         icon="help-circle-outline"
         description="Check our guides about creating and managing LLM endpoints."
```
Comment (nerda-codes) [inline ai-data/managed-inference/index.mdx:35]:
```suggestion
        description="Check our guides about creating and managing Managed Inference endpoints."
```

File: ai-data/managed-inference/index.mdx [hunk 9]
```diff
@@ -20,42 +20,42 @@ meta:
...
         label="View How-tos"
-        url="/ai-data/llm-inference/how-to/"
+        url="/ai-data/managed-inference/how-to/"
     />
     <SummaryCard
         title="Additional content"
         icon="book-open-outline"
         description="Guides to help you choose an LLM endpoint, understand pricing and advanced configuration."
```
Comment (nerda-codes) [inline ai-data/managed-inference/index.mdx:42]:
```suggestion
        description="Guides to help you choose a Managed Inference endpoint, understand pricing and advanced configuration."
```

File: ai-data/managed-inference/index.mdx [hunk 10]
```diff
@@ -20,42 +20,42 @@ meta:
...
-        url="/ai-data/llm-inference/reference-content/"
+        url="/ai-data/managed-inference/reference-content/"
     />
 </Grid>
 
 <ClickableBanner
     productLogo="cli"
-    title="LLM Inference API"
+    title="Managed Inference API"
     description="Learn how to create and manage your Scaleway LLM endpoints through the API."
```
Comment (nerda-codes) [inline ai-data/managed-inference/index.mdx:51]:
```suggestion
    description="Learn how to create and manage your Scaleway Managed Inference endpoints through the API."
```

File: ai-data/managed-inference/quickstart.mdx [hunk 11]
```diff
@@ -1,38 +1,38 @@
...
 
-## How to create an LLM Inference deployment
+## How to create an Managed Inference deployment
```
Comment (nerda-codes) [inline ai-data/managed-inference/quickstart.mdx:29]:
```suggestion
## How to create a Managed Inference deployment
```

File: ai-data/managed-inference/quickstart.mdx [hunk 12]
```diff
@@ -49,11 +49,11 @@ Here are some of the key features of Scaleway LLM Inference:
     </Message>
 6. Click **Create deployment** to launch the deployment process. Once the deployment is ready, it will be listed among your deployments.
 
-## How to access an LLM Inference deployment
+## How to access an Managed Inference deployment
```
Comment (nerda-codes) [inline ai-data/managed-inference/quickstart.mdx:52]:
```suggestion
## How to access a Managed Inference deployment
```

File: ai-data/managed-inference/quickstart.mdx [hunk 13]
```diff
@@ -64,17 +64,17 @@ LLM Inference deployments use dynamic tokens generated with Scaleway's Identity
 
 ## How to interact with an LLM
```
Comment (nerda-codes) [inline ai-data/managed-inference/quickstart.mdx:65]: Should this title still mention LLM?

File: ai-data/managed-inference/reference-content/wizardlm-70b-v1.0.mdx [hunk 14]
```diff
@@ -66,11 +66,11 @@ Make sure to replace `<IAM API key>` and `<Deployment UUID>` with your actual [I
   Ensure that the `messages` array is properly formatted with roles (system, user, assistant) and content.
 </Message>
 
-### Receiving LLM Inference responses
+### Receiving Managed Inference responses
 
-Upon sending the HTTP request to the public or private endpoints exposed by the server, you will receive inference responses from the managed LLM Inference server. 
+Upon sending the HTTP request to the public or private endpoints exposed by the server, you will receive inference responses from the managed Managed Inference server.
```
Comment (nerda-codes) [inline ai-data/managed-inference/reference-content/wizardlm-70b-v1.0.mdx:71]:
```suggestion
Upon sending the HTTP request to the public or private endpoints exposed by the server, you will receive inference responses from the Managed Inference server. 
```

File: ai-data/managed-inference/index.mdx (hunk 8 above)
Comment (nerda-codes) [reply ai-data/managed-inference/index.mdx:35]: Not sure about these

File: compute/gpu/reference-content/choosing-gpu-instance-type.mdx [hunk 15]
```diff
@@ -76,7 +76,7 @@ Remember that there is no one-size-fits-all answer, and the right GPU Instance t
...
-| Better used for                                                     | Image Enconding (8K)                                                                                  | Video Enconding (8K)                                                                                     | 7B LLM Inference                                                                                         | 70B LLM Inference |
+| Better used for                                                     | Image Enconding (8K)                                                                                  | Video Enconding (8K)                                                                                     | 7B Managed Inference                                                                                         | 70B Managed Inference |
```
Comment (bene2k1) [inline compute/gpu/reference-content/choosing-gpu-instance-type.mdx:79]:
```suggestion
| Better used for                                                     | Image Enconding (8K)                                                                                  | Video Enconding (8K)                                                                                     | 7B Managed Inference                                                                                         | 70B LLM Inference |
```

File: ai-data/managed-inference/quickstart.mdx (hunk 13 above)
Comment (bene2k1) [inline ai-data/managed-inference/quickstart.mdx:65]:
```suggestion
## How to interact with Managed Inference
```

File: ai-data/managed-inference/quickstart.mdx (hunk 1 above)
Comment (tgenaitay) [reply ai-data/managed-inference/quickstart.mdx:27]: We can scratch "private" to anticipate a bit as public beta is set to start in July. But we're still beta phase, so specific terms still apply.

**Review by tgenaitay — APPROVED** [review]
LGTM

File: ai-data/managed-inference/quickstart.mdx (hunk 1 above)
Comment (bene2k1) [reply ai-data/managed-inference/quickstart.mdx:27]:
```suggestion
<Message type="important">
  This service is in beta. [Specific terms and conditions](https://www.scaleway.com/en/contracts/) apply.
</Message>
```

**Review by nerda-codes — APPROVED** [review]

**Pull request merged**